workers.py

```diff
@@ -50,6 +50,7 @@ def _retryable(method: str, *args, **kwargs) -> requests.Response:
 def volume_type_validation_worker(
```

Comment:
Maybe rename the worker as well to `topology_worker` or something? Since it's now tied to a certain AI service anymore?

Reply:
It is very much tied to the AI service, right?

It is also calling `AwsVolumeTypeValidation` which is very Volume-Type-Validation specific

So let's keep the name `volume_type_validation_worker` :)

Reply:
Ok, so since this worker is tied to a certain use case, why do we care about setting the `ai_service` dynamically? Do we expect this name to change at all then?

Reply:
The value in `ai_service` is nothing more than a pass-through value, that eventually makes it's way to the Publisher, and then the Upload service.
The change here ensures that the results would be published to `platform.upload.<ai_service>`, so -
`platoform.upload.aiops-volume-type-validation` in this case.

In the end, this is all useful in the Consumer service, useful in the `KAFKA_CONSUMER_TOPIC` context - https://github.com/ManageIQ/aiops-consumer/pull/3